### PR TITLE
update and rename README.md

### PR DESCRIPTION
**Pull request by timt1028**
update and rename 
  1. add Figma url to this repository.
  2. README1111.md to README.md.

### CODE REVIEW

**Review by snowmancc — COMMENTED** [review]
LGTM

**Review by snowmancc — COMMENTED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -16,3 +16,5 @@
   - 後端： Java, Spring Boot
   - 前端： React
 
+citadels-game
+Figma https://www.figma.com/file/9D2QZSsKS1YSVcg6hhA01e/%E5%AF%8C%E9%A5%92%E4%B9%8B%E5%9F%8E?type=design&t=Q5Vwzu5v3Rv1Fuym-1
```
Comment (snowmancc) [inline README.md:20]: 123

**Review by snowmancc — APPROVED** [review]
LGTM

**Review by Irisloverain — COMMENTED** [review]
LGTM

**Review by aprilywy — APPROVED** [review]
LGTM

**Review by Irisloverain — APPROVED** [review]
LGTM

**Pull request merged**